RuleDocumentation/ReservedCmdletChar.md

```diff
@@ -4,9 +4,10 @@
 
 ## Description
 
-You cannot use following reserved characters in a function or cmdlet name as these can cause parsing or runtime errors.
+You cannot use following reserved characters in a function or cmdlet name as these can cause parsing
+or runtime errors.
 
-Reserved Characters include: `#,(){}[]&/\\$^;:\"'<>|?@`*%+=~`
+Reserved Characters include: `#,(){}[]&/\\$^;:\"'<>|?@``*%+=~`
```

Comment:
Will the `` after the @ symbol cause issues with the markdown?

Reply:
Oh just saw comments about ` being used as escape sequence in PS

Reply:
Right. But there is still a markdown rendering issue. I fixed it now.